src/domains/xata/types.d.ts

```diff
@@ -132,4 +132,5 @@ export declare class XataClient extends DatabaseClient<DatabaseSchema> {
   constructor(options?: BaseClientOptions);
 }
 export declare const getXataClient: () => XataClient;
+export declare const xataWorker: any;
```

Comment:
![https://media.tenor.com/sIB-6LgziVIAAAAC/spongebob-squarepants-spongebob.gif](https://media.tenor.com/sIB-6LgziVIAAAAC/spongebob-squarepants-spongebob.gif)

This is a bug 😑🤫

Reply:
Oh, I was indeed a little surprised when there were no types on stream...